ReactSkia/pluginfactory/interfaces/RNSPlatformManagerInterface.cpp

```diff
@@ -13,13 +13,36 @@ namespace rnsplugin {
 
 RNSPlatformManagerInterface::RNSPlatformManagerInterface(PlatformCallbackClient &client)
     :platformManagerName_(format("RNSPlatformManager-{}",nextUniqueId()).str()),
-    platformThread_(!platformManagerName_.empty()? platformManagerName_.c_str(): "RNSPlatformManager-UnknownID"),      //not used for now
-    client_(client) {}
+  platformThread_(!platformManagerName_.empty()? platformManagerName_.c_str(): "RNSPlatformManager-UnknownID"),      //not used for now
+  client_(client) {}
 
 RNSPlatformManagerInterface::~RNSPlatformManagerInterface(){
   // platformThread_ will  flush the all theee pending tasks in thread before exiting.
 }
 
+RNSPlatformManagerInterface::PlatformDevice::DeviceProcessUintInfo::DeviceProcessUintInfo(string lname, string lcores, string ltechnology, string lfrequency)
+    :name(lname),
+  cores(lcores),
+  technology(ltechnology),
+  frequency(lfrequency){}
+
+RNSPlatformManagerInterface::PlatformDevice::DeviceOsInfo::DeviceOsInfo(string osProvider, string osName, string osVersion)
+    : provider(osProvider),
+  name(osName),
+  version(osVersion){}
+
+RNSPlatformManagerInterface::PlatformDevice::DeviceHardwareInfo::DeviceHardwareInfo(string hwManufacturer, string hwModel, string hwType)
+    :manufacturer(hwManufacturer),
+  model(hwModel),
+  type(hwType){}
+
+RNSPlatformManagerInterface::PlatformDevice:: DeviceScreenInfo::DeviceScreenInfo(string screenDepth, int screenWidth, int screenHeight, string screenRefresh)
```

Comment:
suggest screenRefreshRate over screenRefresh. If you are changing change it associated places and assignments as well

Reply:
@Naveen33-am  you can change it